common/search/src/main/scala/weco/api/search/elasticsearch/ElasticsearchService.scala

```diff
@@ -146,9 +145,9 @@ class ElasticsearchService(elasticClient: ElasticClient)(
             }
             transaction.setLabel("elasticTookTotal", finalTotalTimeTaken)
 
-            (finalErrors, finalSearchResponses)
+            finalResults
 
-          case Left(err) => (Seq(ElasticsearchError(err)), Seq.empty)
+          case Left(err) => throw err.asException
```

Comment:
Is an unhandled throw safe here? We'll catch it and wrap it in a Future?

Reply:
This is wrapped in the future started by `elasticClient.execute(request)` - it's difficult to see at the end of match statement. I'll move this to the top so it's easier to see this is safe.